Makefile

```diff
@@ -42,17 +45,28 @@ bin/%:
 generate:
 	@hack/codegen.sh
 
-deploy: kind-load-manager
+deploy:
+	# We need to make sure the namespace is created before we apply any namespace-scoped configurations into cluster.
+	kubectl create namespace bulward-system
```

Comment:
We need to create the namespace first before we apply any configurations to the cluster, otherwise namespace-scoped objects cannot be created.